layout-server/src/test/java/org/finos/vuu/layoutserver/model/LayoutTest.java

```diff
@@ -0,0 +1,27 @@
+package org.finos.vuu.layoutserver.model;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
```

Comment:
IntelliJ pro-tip: hit `ctrl + alt + o` to optimise your imports. There's also a setting to do this automatically every time you commit.

![image](https://github.com/ScottLogic/finos-vuu/assets/79100986/8df5e52a-e4b2-4dd8-b6a6-439031cae3d5)

Reply:
that's very handy! Thanks Peter :)

Reply:
done!